examples/templates/README.md

```diff
@@ -30,14 +54,22 @@ In order to create new code examples with these templates, each new example must
     "url": "https://github.com/handsontable/handsontable/issues"
   },
   "author": "Handsoncode <hello@handsontable.com>",
-  "license": "MIT",
+  "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "handsontable": "latest",
-    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks
+    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks, e.g. "@handsontable/react": "latest"
   }
 }
 ```
 
-#### Don't touch
```

Comment:
Oh my, it sounds too aggressive to me. Can we change it to sth like "Package.json properties" or "Package.json scripts"?

Reply:
This line has been removed already 😄